### DISCUSSION TITLE
How do I change the GF skin? (Read below for more context)

### DISCUSSION THREAD

**Discussion by Megalonumber0ne** [YoshiCrafter29/YoshiCrafterEngine#114] [Q&A]
So, I am trying to port Doki Doki Takeover to Yoshi Engine and am having trouble learning how to change how GF looks ingame, I have her character added with animations and everything but I don't know how to get her into a song.
Screenshot for whatever it's worth:
![image](https://user-images.githubusercontent.com/87905744/159166930-6f4e0695-d68d-4fff-8100-387117543834.png)

**Comment by Megalonumber0ne** [comment]
For whatever else it's worth I am using Version 1.7.0 right now

**Reply by BernardoGP4504** [reply]
go into chart editor and change gf

**Reply by SrtHero278** [reply]
if he's still on 1.7, that wont work.

**Comment by SrtHero278** [comment]
![image](https://user-images.githubusercontent.com/93018286/175791698-a319bc3b-a55e-43d9-b45a-a74a99a4ae5d.png)
you can set the gf version on the scripts.
